release_notes/41-release-notes.adoc

```diff
@@ -12,15 +12,16 @@ toc::[]
 [options="header"]
 |====
 |{product-title-short} version |Released on
-|`4.1` |29 Jun 2023
+|`4.1` |29 June 2023
+|`4.1.1` |11 July 2023
 |====
 
 [id="about-this-release_{context}"]
 == About this release
 
 [IMPORTANT]
 ====
-If you are using {ocp} and the Operator for upgrades, see xref:../release_notes/41-release-notes.adoc#kernel-module-collection-method_release-notes-41[Kernel module collection method] for important information and steps you must take before upgrading.
+If you are using {ocp} and the {product-title-short} Operator, and your upgrades are failing, see link:https://access.redhat.com/solutions/7023432[RHACS Operator upgrade to 4.1.0 stuck in Pending state due removal of "KernelModule" collection method] for important information and steps you must take to resolve those issues.
```

Comment:
I know the link text includes the title of another topic, but shouldn't there be something after "due," such as ""to?"